### PR TITLE
update ice shelf files

### PR DESCRIPTION
**Pull request by mark-petersen**
This only effects simulations with ice shelves.  

- Update default flags for ice shelf melt fluxes.
- Change EC60to30v3wLI IC to be 60 layer.
- Correct RRS30to10v3wLI salinity restoring to be 12-monthly.

### CODE REVIEW

**Comment by mark-petersen** [comment]
@xylar please double-check my values, and let me know if there are other ice shelf specific changes that should be made here.

File: components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml [hunk 1]
```diff
@@ -275,11 +276,11 @@
 <config_land_ice_flux_boundaryLayerNeighborWeight>0.0</config_land_ice_flux_boundaryLayerNeighborWeight>
 <config_land_ice_flux_cp_ice>2.009e3</config_land_ice_flux_cp_ice>
 <config_land_ice_flux_rho_ice>918</config_land_ice_flux_rho_ice>
-<config_land_ice_flux_topDragCoeff>2.5e-3</config_land_ice_flux_topDragCoeff>
+<config_land_ice_flux_topDragCoeff>4.48e-3</config_land_ice_flux_topDragCoeff>
```
Comment (xylar) [inline components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml]: @mark-petersen, I believe these should only be the new defaults for EC60to30wLI.  The old values are appropriate for RRS30to10.  If I understand the syntax correctly, this should be:
```
<config_land_ice_flux_topDragCoeff>2.5e-3</config_land_ice_flux_topDragCoeff>
<config_land_ice_flux_topDragCoeff ocn_grid="oEC60to30v3wLI">4.48e-3</config_land_ice_flux_topDragCoeff>
...
<config_land_ice_flux_jenkins_heat_transfer_coefficient>0.011</config_land_ice_flux_jenkins_heat_transfer_coefficient>
<config_land_ice_flux_jenkins_heat_transfer_coefficient ocn_grid="oEC60to30v3wLI">0.00295</config_land_ice_flux_jenkins_heat_transfer_coefficient>
<config_land_ice_flux_jenkins_salt_transfer_coefficient>3.1e-4</config_land_ice_flux_jenkins_salt_transfer_coefficient>
<config_land_ice_flux_jenkins_salt_transfer_coefficient ocn_grid="oEC60to30v3wLI">8.42e-5</config_land_ice_flux_jenkins_salt_transfer_coefficient>
```

These values are correct.

**Comment by mark-petersen** [comment]
Rebased in origin/master, including updates to edison modules in #2377

**Comment by mark-petersen** [comment]
@xylar thanks for your changes above.  They are in the previous commit.  I'm testing this branch now.

**Comment by rljacob** [comment]
@mark-petersen what is the status of this PR?

**Comment by rljacob** [comment]
@mark-petersen what is the status of this PR?

**Comment by mark-petersen** [comment]
Testing it now with @xylar.  Will be ready to merge next week.

File: components/mpas-ocean/cime_config/buildnml [hunk 2]
```diff
@@ -103,8 +103,8 @@ def buildnml(case, caseroot, compname):
         decomp_date += '160830'
         decomp_prefix += 'mpas-o.graph.info.'
     elif ocn_grid == 'oEC60to30v3wLI':
-        ic_date += '170328'
-        ic_prefix += 'oEC60to30v3wLI'
+        ic_date += '171031'
+        ic_prefix += 'oEC60to30v3wLI60lev'
```
Comment (mark-petersen) [inline components/mpas-ocean/cime_config/buildnml]: On master already, remove from this PR.

File: components/mpas-ocean/cime_config/buildnml [hunk 3]
```diff
@@ -158,7 +158,7 @@ def buildnml(case, caseroot, compname):
         ic_prefix += 'oRRS30to10v3wLI'
         decomp_date += '171109'
         decomp_prefix += 'mpas-o.graph.info.'
-        restoring_file += 'sss.monthlyClimatology.PHC2_salx_040803.oRRS30to10v3wLI.nc'
+        restoring_file += 'sss.monthlyClimatology.PHC2_salx.2004_08_03.180503.nc'
```
Comment (mark-petersen) [inline components/mpas-ocean/cime_config/buildnml]: stays

File: components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml [hunk 4]
```diff
@@ -280,10 +281,13 @@
 <config_land_ice_flux_cp_ice>2.009e3</config_land_ice_flux_cp_ice>
 <config_land_ice_flux_rho_ice>918</config_land_ice_flux_rho_ice>
 <config_land_ice_flux_topDragCoeff>2.5e-3</config_land_ice_flux_topDragCoeff>
+<config_land_ice_flux_topDragCoeff ocn_grid="oEC60to30v3wLI">4.48e-3</config_land_ice_flux_topDragCoeff>
 <config_land_ice_flux_ISOMIP_gammaT>1e-4</config_land_ice_flux_ISOMIP_gammaT>
 <config_land_ice_flux_rms_tidal_velocity>5e-2</config_land_ice_flux_rms_tidal_velocity>
 <config_land_ice_flux_jenkins_heat_transfer_coefficient>0.011</config_land_ice_flux_jenkins_heat_transfer_coefficient>
+<config_land_ice_flux_jenkins_heat_transfer_coefficient ocn_grid="oEC60to30v3wLI">0.00295</config_land_ice_flux_jenkins_heat_transfer_coefficient>
 <config_land_ice_flux_jenkins_salt_transfer_coefficient>3.1e-4</config_land_ice_flux_jenkins_salt_transfer_coefficient>
+<config_land_ice_flux_jenkins_salt_transfer_coefficient ocn_grid="oEC60to30v3wLI">8.42e-5</config_land_ice_flux_jenkins_salt_transfer_coefficient>
```
Comment (mark-petersen) [inline components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml]: These changes will be kept in the next PR.

**Comment by mark-petersen** [comment]
The changes in this PR involving runoff from Antartica are now defunct, as @darincomeau and @jonbob are making a version with data iceberg calving.  The remaining parts summed up to a total of 5 lines, which I have just added to https://github.com/E3SM-Project/E3SM/tree/jonbob/scripts/data-iceberg-support as 05a62bbba836cd83788fbeb74dd742a8138d09f1 and c04cb8120ccbab1766218e02e32164927a5f2608.  We all agreed it was simpler to close this PR, and thoroughly test the next one instead.

**Pull request closed without merging**